timemachine/cpp/src/kernels/k_exchange.cu

```diff
@@ -33,15 +33,21 @@ k_copy_batch<double>(const int N, const int batch_size, const double *__restrict
 // indices for each sample. Note that the output mol offsets are constructed so that the start of the mol is
 // the starting atom idx rather than the prefix sum of mol lengths that the mol_offsets is.
 void __global__ k_setup_proposals(
-    const int batch_size,                      // Number of molecules to setup
-    const int num_atoms_in_each_mol,           // number of atoms in each sample
+    const int total_proposals,
+    const int batch_size,            // Number of molecules to setup
```

Comment:
comment here seems wrong

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1274/commits/b80d8d181a8da212087f4c07778dbf56a9f241d4